examples/headless-example/src/components/App.svelte

```diff
@@ -79,15 +92,32 @@
     createWorkflowsQuery({
       select: workflows => {
         return Array.isArray(workflows)
-          ? workflows?.find(
-              workflow => workflow?.workflowDefinition?.name === import.meta.env.VITE_EXAMPLE_TYPE,
-            )
+          ? workflows
```

Comment:
Suggestion: filter by active, merge (active and non active), sort by resolved time, find